cluster/manager/manager.go

```diff
@@ -492,6 +546,15 @@ func (c *ClusterManager) watchDB(key string, opaque interface{},
 
 	c.gossip.UpdateCluster(c.getNonDecommisionedPeers(db))
 
+	if len(db.ClusterDomains.ActiveMap) > 0 {
+		// Copy over the active map
+		c.activeMap = make(types.ClusterDomainsActiveMap)
```

Comment:
(below when starting gossip we don't make a copy, I would suggest gossip always making a copy of this instead of osd passing a copy).

Reply:
gossip always makes a copy of the active map received through the API.
I wanted to make a copy here as well, because DB is being used elsewhere in this function as well.